lib/winrm/connection_opts.rb

```diff
@@ -68,8 +68,8 @@ def validate_required_fields
       if self[:client_cert]
         raise 'path to client key is required' unless self[:client_key]
       else
-        raise 'user is a required option' unless self[:user]
-        raise 'password is a required option' unless self[:password]
+        raise 'user is a required option' unless self[:transport] == :kerberos 

```

Comment:
with this change: if user and password but not kerberos is provided, this will raise, won't it? that seems incorrect.